cmd/skopeo/sync.go

```diff
@@ -104,6 +105,7 @@ See skopeo-sync(1) for details.
 	flags.StringVarP(&opts.destination, "dest", "d", "", "DESTINATION transport type")
 	flags.BoolVar(&opts.scoped, "scoped", false, "Images at DESTINATION are prefix using the full source image path as scope")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Copy all images if SOURCE-IMAGE is a list")
+	flags.BoolVar(&opts.dry, "dry", false, "Run without actually copying data")
```

Comment:
Maybe `"dry-run"`? (Disclaimer: not a native speaker); AFAIK the idiom is “dry run”, and “dry“ does not usually have the “test-only” meaning outside of that idiom.

(I know, naming is hard and colors of the bike-shed…)

Reply:
I agree with you, this makes total sense.